frontend/pages/admin/AppSettingsPage/AppSettingsPage.tsx

```diff
@@ -10,15 +13,47 @@ import deepDifference from "utilities/deep_difference";
 import { IConfig } from "interfaces/config";
 import { IApiError } from "interfaces/errors";
 
-// @ts-ignore
-import AppConfigForm from "components/forms/admin/AppConfigForm";
+import PATHS from "router/paths";
+import Info from "./cards/Info";
+import WebAddress from "./cards/WebAddress";
+import Sso from "./cards/Sso";
+import Smtp from "./cards/Smtp";
+import AgentOptions from "./cards/Agents";
+import HostStatusWebhook from "./cards/HostStatusWebhook";
+import Statistics from "./cards/Statistics";
+import Advanced from "./cards/Advanced";
+
+interface IAppSettingsPageProps {
+  params: Params;
+}
 
 export const baseClass = "app-settings";
 
-const AppSettingsPage = (): JSX.Element => {
+const AppSettingsPage = ({
+  params: { section: sectionTitle },
+}: IAppSettingsPageProps): JSX.Element => {
   const { renderFlash } = useContext(NotificationContext);
   const { setConfig } = useContext(AppContext);
 
+  const [showInfo, setShowInfo] = useState<boolean>(sectionTitle === "info");
+  const [showWebAddress, setShowWebAddress] = useState<boolean>(
+    sectionTitle === "webaddress"
+  );
+  const [showSso, setShowSso] = useState<boolean>(sectionTitle === "sso");
+  const [showSmtp, setShowSmtp] = useState<boolean>(sectionTitle === "smtp");
+  const [showAgents, setShowAgents] = useState<boolean>(
+    sectionTitle === "agents"
+  );
+  const [showHostStatusWebhook, setShowHostStatusWebhook] = useState<boolean>(
+    sectionTitle === "host-status-webhook"
+  );
+  const [showStatistics, setShowStatistics] = useState<boolean>(
+    sectionTitle === "statistics"
+  );
+  const [showAdvanced, setShowAdvanced] = useState<boolean>(
+    sectionTitle === "advanced"
+  );
+
```

Comment:
What if instead of a state variable for each section, there was a single state that tracked which section was selected? For example: 

```javascript
const [activeSection, setActiveSection] = useState<string>("info");
```